### PR TITLE
chore: add switch themes apps

### PR DESCRIPTION
**Pull request by yanmariomenev**
Added projects that showcase how to switch between Kendo themes, such as light and dark.

### CODE REVIEW

File: examples-standalone/switch-kendo-themes/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+This project demonstrates various approaches to dynamically switching Kendo themes in an Angular application, enabling users to choose their preferred visual styles without reloading the page. The implementations showcase the following methods:
+
+- CSS Theme Switching
+- SCSS Theme Switching
+- SCSS Theme Switching using Angular Services
+- Using CDN links for Kendo UI themes
+
+## Implementations
+
+### 1. CSS Theme Switching
```
Comment (mbechev) [inline examples-standalone/switch-kendo-themes/README.md:16]: Is there any reason leaving the numbers here?

File: examples-standalone/switch-kendo-themes/css-switch-themes/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# ScssProject
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 18.0.6.
+
+## Development server
```
Comment (mbechev) [inline examples-standalone/switch-kendo-themes/css-switch-themes/README.md:5]: `npm install` is missing as a required command for the project.

**Review by mbechev — COMMENTED** [review]
The code of the projects looks good to me.

**Comment by yanmariomenev** [comment]
I will close this. The applications need some additional polish, especially the CSS one, and a full article along with them would be better.

**Pull request closed without merging**